src/bitmessageqt/foldertree.py

```diff
@@ -107,44 +114,63 @@ def defaultLabel(self):
         elif self.address is None or self.type == AccountMixin.ALL:
             return unicode(
                 str(_translate("MainWindow", "All accounts")), 'utf-8')
-        if retval is None:
-            return unicode(self.address, 'utf-8')
-        else:
-            return retval
 
+        return retval or unicode(self.address, 'utf-8')
```

Comment:
I'm not sure this is correct. I think it returns a boolean which is either true or false.

Reply:
It probably should be more like `return retval if retval is not None else unicode(self.address, 'utf-8')`

Reply:
It will return False only is `unicode(self.address, 'utf-8')` is False. But well it should be tested off course. That `foldertree` module is really essential for ui operation.

Reply:
You can try yourself:

In [2]: def test_or_return(foo, bar):
   ...:     return foo or bar
   ...: 

In [3]: test_or_return(1, 2)
Out[3]: 1

In [4]: test_or_return(None, 2)
Out[4]: 2

In [5]: test_or_return('', 2)
Out[5]: 2

In [6]: test_or_return('', '')
Out[6]: ''

In [7]: test_or_return('', False)
Out[7]: False

Reply:
ok